zstacklib/zstacklib/utils/qemu_nbd.py

```
@@ -0,0 +1,21 @@
+from zstacklib.utils import linux
+from zstacklib.utils import shell
+import subprocess
+
+
+def export(port, *args):
+    command = 'qemu-nbd -p %s' % port
+    if args:
+        command += ' ' + ' '.join(str(arg) for arg in args)
+    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Ruff (0.14.7)

`subprocess` call with `shell=True` identified, security issue

(S602)